cmd/tracee-rules/output.go

```diff
@@ -104,3 +117,47 @@ func sendToWebhook(t *template.Template, res detect.Finding, webhook string, web
 	_ = resp.Body.Close()
 	return nil
 }
+
+func sendToFluentForward(t *template.Template, res detect.Finding, fluentForwardUrl string, fluentForwardTemplate string, fluentForwardTag string) error {
+	if fluentForwardTag == "" {
+		return errors.New("error sending to Forward: --forward-tag flag is required when using --forward-url flag")
+	}
+
+	// Convert to payload in appropriate format
+	var payload string
+
+	switch {
+	case fluentForwardTemplate != "":
+		if t == nil {
+			return fmt.Errorf("error writing to template: template not initialized")
+		}
+		buf := bytes.Buffer{}
+		if err := t.Execute(&buf, res); err != nil {
+			return fmt.Errorf("error writing to the template: %v", err)
+		}
+		payload = buf.String()
+	default:
+		return errors.New("error sending to Forward: --forward-template flag is required when using --forward-url flag")
+	}
```

Comment:
```
if fluentForwardTemplate == "" {
     return errors.New("error sending to Forward: --forward-template flag is required when using --forward-url flag")
}

buf := bytes.Buffer{}
if err := t.Execute(&buf, res); err != nil {
	return fmt.Errorf("error writing to the template: %v", err)
}
```

Reply:
I agree this is more readable but then the webhook function does it this way so I wanted to stay consistent.